HW6/Runner.py

```diff
@@ -0,0 +1,25 @@
+from HW6.Num_n_Square import char_count
+from HW6.Num_n_Square import num_n_square
+from HW6.MoreCountedWordFinder import more_counted_word
+
+
+def runner(inner_lst):
```

Comment:
there is more generic way using getattr. in your approach we can work only with this particular functions